### PR TITLE
Small Code Improvements

### PR DESCRIPTION
**Pull request by Sigmanificient**
Hi, I've seen in your `CONTRIBUTING.md` that you are welcome to code improvements:

> - Feel free to contribute by cleaning up the code.
> - Fix my bad code (there's lots of it.)

So i did some little to improve the code readability, mostly reformatting and spacing

### CODE REVIEW

**Comment by LewdNeko** [comment]
Need some time to go over this thoroughly.

**Comment by Sigmanificient** [comment]
> 
> 
> Need some time to go over this thoroughly.

sure, no problems!

File: pypresence/presence.py [hunk 1]
```diff
@@ -55,20 +78,43 @@ class AioPresence(BaseClient):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs, isasync=True)
```
Comment (LewdNeko) [inline pypresence/presence.py:79]: Looks like you forgot to fully refactor `isasync` to `is_async`.

File: tests/basic_presence_test.py [hunk 2]
```diff
@@ -17,10 +16,14 @@
     "Eighty percent of success is showing up.",
     "Life is what happens to you while you’re busy making other plans.",
     "Strive not to be a success, but rather to be of value.",
-    "The best time to plant a tree was 20 years ago. The second best time is now.",
+    (
+        "The best time to plant a tree was 20 years ago."
+        " The second best time is now."
+    ),
```
Comment (LewdNeko) [inline tests/basic_presence_test.py:22]: What's going on here? Was the line too ling?

File: examples/rich-presence-loop.py [hunk 3]
```diff
@@ -16,11 +16,18 @@
     "Eighty percent of success is showing up.",
     "Life is what happens to you while you’re busy making other plans.",
     "Strive not to be a success, but rather to be of value.",
-    "The best time to plant a tree was 20 years ago. The second best time is now.",
+    (
+        "The best time to plant a tree was 20 years ago." 
+        "The second best time is now."
```
Comment (LewdNeko) [inline examples/rich-presence-loop.py:21]: Ditto to below comment.

File: pypresence/client.py [hunk 4]
```diff
@@ -18,37 +18,51 @@ def __init__(self, *args, **kwargs):
...
         del self._events[event]
 
     # noinspection PyProtectedMember
     def on_event(self, data):
         if self.sock_reader._eof:
             raise PyPresenceException('feed_data after feed_eof')
+
         if not data:
             return
+
```
Comment (LewdNeko) [inline pypresence/client.py:50]: Honestly not sure if these newlines across this feel are really necessary, seems like personal preference but oh well, doesn't bother me too much.

**Review by LewdNeko — CHANGES_REQUESTED** [review]
Just a couple minor nags and should be grand :)

File: tests/basic_presence_test.py (hunk 2 above)
Comment (Sigmanificient) [reply tests/basic_presence_test.py:22]: Yep, according to `pep8` & `E501` code should be restricted as `80` characters (72 for docstring & comment).
This is getting less popular nowadays and people more often tend to go for `90`-`100` or even `120` as max length.
(i was used to `120` for a long but switched to `80`, allowing me to always have 2 python files open in split mode)

File: pypresence/client.py (hunk 4 above)
Comment (Sigmanificient) [reply pypresence/client.py:50]: I like having space between block help for readability, can be removed it you prefer tho

Comment (LewdNeko) [reply pypresence/client.py:50]: It's just that for me it does little to improve readability. Entirely a preference thing, it's fine to keep it :)

File: tests/basic_presence_test.py (hunk 2 above)
Comment (LewdNeko) [reply tests/basic_presence_test.py:22]: It's less the choice of line length, more so the use of `(\n""\n""),` - I've never seen this way of splitting up a string. My go-to is `""\n\\n"",`

**Comment by LewdNeko** [comment]
I'll wait until the checks are working correctly before merging :)

**Comment by Sigmanificient** [comment]
> 
> 
> I'll wait until the checks are working correctly before merging :)

The tox checker fail cause it is missing the os key `PYPRESENCE_CLIENT_ID`
```py
 Traceback (most recent call last):
  File "tests/basic_presence_test.py", line 6, in <module>
    client_id = os.environ["PYPRESENCE_CLIENT_ID"]
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/os.py", line 681, in __getitem__
    raise KeyError(key) from None
KeyError: 'PYPRESENCE_CLIENT_ID'
```

File: tests/basic_presence_test.py (hunk 2 above)
Comment (Sigmanificient) [reply tests/basic_presence_test.py:22]: > 
> 
> It's less the choice of line length, more so the use of `(\n""\n""),` - I've never seen this way of splitting up a string. My go-to is `""\n\\n"",`

The problem with using `\`, in my opinion, is that it offset the whole text depending on the variable name, wish can be a pain to handle when having a long one, see below:
```py
long_text_variable_name = "Lorem ipsum dolor sit amet, consectetur adipiscing" \
                          " elit, sed do eiusmod tempor incididunt ut labore" \
                          " et dolore magna aliqua. Ut enim ad minim veniam," \
                          " quis nostrud exercitation ullamco laboris nisi ut" \
                          " aliquip ex ea commodo consequat. Duis aute irure" \
                          " dolor in reprehenderit in voluptate velit esse" \
                          " cillum dolore eu fugiat nulla pariatur. Excepteur" \
                          " sint occaecat cupidatat non proident, sunt in" \
                          " culpa qui officia deserunt mollit anim id est" \
                          " laborum."
```

```py
long_text_variable_name = (
    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod "
    "tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim "
    "veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea "
    "commodo consequat. Duis aute irure dolor in reprehenderit in voluptate "
    "velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat "
    "cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id "
    "est laborum."
)
```

Aslo python totally understand the string contatenation 
```py
>>> "a" "b"
'ab'
```
and directly optimize it within the compilation phase (a very tiny phase before the interpretation that checks syntax, and optimize some part such as string, ...)

**Review by LewdNeko — APPROVED** [review]

**Comment by LewdNeko** [comment]
Hey there, sorry that it's taken so long but unfortunately since then have made these changes into conflicts. I'm going to close it for now but you're more than welcome to reopen it if you want.

**Pull request closed without merging**